src/pybamm_cookiecutter/cli.py

```diff
@@ -0,0 +1,60 @@
+import copier
+import argparse
+from pathlib import Path
+from colorama import Fore
+import os
+
+project_root = Path(__file__).resolve().parent.parent
+TEMPLATE = str(project_root)
+
+def pybamm_cookiecutter_cli():
+    """
+    Command Line Interface (CLI) for generating PyBaMM based projects using copier.
+
+    Parameters
+    ----------
+    --path: pathlib.Path
+
+    Examples
+    -------
+    $ pybamm-cookiecutter
+    Generates a project in the current working directory of the terminal.
+    $ pybamm-cookiecutter --path /myproject
+    Generates a project in the `myproject` directory.
+    """
+    try:
+        parser = argparse.ArgumentParser(description = "A copier template generator for PyBaMM based projects")
+        parser.add_argument(
+            "--path", type = str,
+            required = False,
+            default = os.getcwd(),
+            help = "The destination path for project generation. The default is the current working directory"
+        )
+
+        from pybamm_cookiecutter import __version__ as version
+        parser.add_argument(
+            '--version',
+            action='version',
+            version=f'PyBaMM Cookiecutter CLI Version - {version}'
+        )
+
+        parser.add_argument(
+            "--defaults",
+            action="store_true",
+            help="Whether to use default options for generating the template"
```

Comment:
Something like `pybamm-cookiecutter help` should be helpful, is it already there?

Reply:
if not can u add it?

Reply:
It's already there:

```bash
❯ pybamm-cookiecutter --help
```

```
usage: pybamm-cookiecutter [-h] [--path PATH] [--version] [--defaults]

A copier template generator for PyBaMM based projects

options:
  -h, --help   show this help message and exit
  --path PATH  The destination path for project generation. The default is the current working directory
  --version    show program's version number and exit
  --defaults   Whether to use default options for generating the template
```